cpp/src/barretenberg/stdlib/primitives/bool/bool.test.cpp

```diff
@@ -1,86 +1,74 @@
 #include "bool.hpp"
-#include "barretenberg/plonk/proof_system/constants.hpp"
+#include "barretenberg/plonk/composer/standard_composer.hpp"
+#include <gtest/gtest-typed-test.h>
 #include <gtest/gtest.h>
-// #include <plonk/composer/standard_composer.hpp>
+#include <gtest/internal/gtest-type-util.h>
```

Comment:
What is this needed for?

Reply:
Could be the formatter added this and you didn't even mean to, might not actually be needed though.

Reply:
yeeah it did, i think it can be removed